### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [madebyconnor/change#5]
The user @carlsonsantana validate your site template "https://madebyconnor.github.io/change/" and found these accessibility errors:
* Page https://madebyconnor.github.io/change/:
  * Issue *fb9afb6a-2d00-47d2-bab5-eafe5dfb8c34*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.3:1. Recommendation: change text colour to #73777b.
    * **Selector**: `html > body > div > footer > div > h6 > a`
    * **Context**: `<a href="https://twitter.com/connor_baer" aria-label="Follow Connor Bär on Twitter.">Connor Bär</a>`
  * Issue *d227c320-030a-4c73-be5e-25f086e63563*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.3:1. Recommendation: change text colour to #73777b.
    * **Selector**: `html > body > div > footer > div > h6`
    * **Context**: `<h6 class="footer-text">© 2017 <a href="https://twitter...</h6>`
  * Issue *d62f4e14-0e51-450c-9007-5a0789281cb8*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.32:1. Recommendation: change text colour to #6f777f.
    * **Selector**: `html > body > div > footer > div > div:nth-child(2) > div:nth-child(2) > nav > ul > li:nth-child(3) > a`
    * **Context**: `<a href="http://eating.connorbaer.co/" target="_blank" rel="noopener noreferrer" aria-label="Eating">Eating</a>`
  * Issue *cca91178-2e0f-4018-b7ab-08722c6c1883*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.32:1. Recommendation: change text colour to #6f777f.
    * **Selector**: `html > body > div > footer > div > div:nth-child(2) > div:nth-child(2) > nav > ul > li:nth-child(2) > a`
    * **Context**: `<a href="http://connorbaer.co/writing" target="_blank" rel="noopener noreferrer" aria-label="Writing">Writing</a>`
  * Issue *39f18875-8aed-42d9-b7d7-1f2506b7d1b7*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.32:1. Recommendation: change text colour to #6f777f.
    * **Selector**: `html > body > div > footer > div > div:nth-child(2) > div:nth-child(2) > nav > ul > li:nth-child(1) > a`
    * **Context**: `<a href="http://connorbaer.co/creating" target="_blank" rel="noopener noreferrer" aria-label="Creating">Creating</a>`
  * Issue *2204af27-52a0-4b09-8b90-425f50eee9d0*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.32:1. Recommendation: change text colour to #6f777f.
    * **Selector**: `html > body > div > footer > div > div:nth-child(2) > div:nth-child(1) > nav > ul > li:nth-child(3) > a`
    * **Context**: `<a href="https://github.com/madebyconnor/change" aria-label="GitHub">GitHub</a>`
  * Issue *4ed0c226-5df9-49ed-80ae-a02e9220fd29*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.32:1. Recommendation: change text colour to #6f777f.
    * **Selector**: `html > body > div > footer > div > div:nth-child(2) > div:nth-child(1) > nav > ul > li:nth-child(2) > a`
    * **Context**: `<a class="" href="/change/about/" aria-label="About">About</a>`
  * Issue *cb6a91f3-db62-4a8b-8fc1-08ac716099a6*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.32:1. Recommendation: change text colour to #6f777f.
    * **Selector**: `html > body > div > footer > div > div:nth-child(2) > div:nth-child(1) > nav > ul > li:nth-child(1) > a`
    * **Context**: `<a class="" href="/change/blog/" aria-label="Blog">Blog</a>`
  * Issue *ebe5305a-4c70-4f7e-a6b7-de1781b291e6*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.32:1. Recommendation: change text colour to #6f777f.
    * **Selector**: `#posts > article > div:nth-child(11) > span`
    * **Context**: `<span class="meta-text">&nbsp;·&nbsp;&nbsp; 2 min read</span>`
  * Issue *9459532a-c7d5-46c4-8fc9-3bc671cc64bf*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.32:1. Recommendation: change text colour to #6f777f.
    * **Selector**: `#posts > article > div:nth-child(11) > time`
    * **Context**: `<time class="meta-text" datetime="2013-09-03T00:00:00+00:00">September 03, 2013</time>`
  * Issue *346884dc-26d1-4b60-b626-468663a3e295*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.32:1. Recommendation: change text colour to #6f777f.
    * **Selector**: `#posts > article > div:nth-child(8) > span`
    * **Context**: `<span class="meta-text">&nbsp;·&nbsp;&nbsp; 2 min read</span>`
  * Issue *f59e4c52-972d-4741-a13e-e38281084c49*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.32:1. Recommendation: change text colour to #6f777f.
    * **Selector**: `#posts > article > div:nth-child(8) > time`
    * **Context**: `<time class="meta-text" datetime="2013-10-25T00:00:00+00:00">October 25, 2013</time>`
  * Issue *df1595e9-d585-451c-9ee9-20636ab5a991*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.32:1. Recommendation: change text colour to #6f777f.
    * **Selector**: `#posts > article > div:nth-child(5) > span`
    * **Context**: `<span class="meta-text">&nbsp;·&nbsp;&nbsp; 1 min read</span>`
  * Issue *46470681-036b-42ad-b3c5-e05c5ab4965f*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.32:1. Recommendation: change text colour to #6f777f.
    * **Selector**: `#posts > article > div:nth-child(5) > time`
    * **Context**: `<time class="meta-text" datetime="2015-03-25T00:00:00+00:00">March 25, 2015</time>`
  * Issue *f6d124bf-b3ca-4586-b74f-f254bbaa96fe*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.32:1. Recommendation: change text colour to #6f777f.
    * **Selector**: `#posts > article > div:nth-child(2) > span`
    * **Context**: `<span class="meta-text">&nbsp;·&nbsp;&nbsp; 1 min read</span>`
  * Issue *9a4650bd-631f-47f5-aed1-48d7e6b9e32e*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.32:1. Recommendation: change text colour to #6f777f.
    * **Selector**: `#posts > article > div:nth-child(2) > time`
    * **Context**: `<time class="meta-text" datetime="2015-09-05T00:00:00+00:00">September 05, 2015</time>`
  * Issue *da6f4ef4-2385-487b-9269-9322f43bd666*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.32:1. Recommendation: change text colour to #6f777f.
    * **Selector**: `html > body > div > article > header > div:nth-child(2) > span`
    * **Context**: `<span class="meta-text">Share me!</span>`
  * Issue *be75610f-8815-4006-9a08-cce61afe4a88*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.32:1. Recommendation: change text colour to #6f777f.
    * **Selector**: `#js-metabarNav > ul > li:nth-child(3) > a`
    * **Context**: `<a href="https://github.com/madebyconnor/change" aria-label="GitHub">GitHub</a>`
  * Issue *85a67d4b-8b4c-486a-8567-95f3b4b111d2*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.32:1. Recommendation: change text colour to #6f777f.
    * **Selector**: `#js-metabarNav > ul > li:nth-child(2) > a`
    * **Context**: `<a class="" href="/change/about/" aria-label="About">About</a>`
  * Issue *613365de-1597-4653-8573-7ee21bf888cf*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.32:1. Recommendation: change text colour to #6f777f.
    * **Selector**: `#js-metabarNav > ul > li:nth-child(1) > a`
    * **Context**: `<a class="" href="/change/blog/" aria-label="Blog">Blog</a>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
You can view the [full validation results in our website](https://wapyce.herokuapp.com/validation/validations/8d011e20-cb7e-4f92-99bf-5bdddcf19ecd/).

**Comment by connor-baer** [comment]
I no longer actively maintain this theme, but I'm open to PRs!
